dropwizard-jackson/src/test/java/io/dropwizard/jackson/JacksonDeserializationOfBigNumbersToDurationTest.java

```diff
@@ -0,0 +1,100 @@
+package io.dropwizard.jackson;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import javax.annotation.Nullable;
+import java.time.Duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class JacksonDeserializationOfBigNumbersToDurationTest {
+
+    private final ObjectMapper objectMapper = Jackson.newObjectMapper();
+
+    @Test(timeout = 5000)
+    public void testDoesNotAttemptToDeserializeExtremelyBigNumbers() {
+        assertThatExceptionOfType(JsonMappingException.class).isThrownBy(
```

Comment:
Should this test also confirm that the exception message contains something about the value being out of range?

Reply:
Absolutely. I've added check for the exception message here and in the duration test case.